demo/spoke/install-config.yaml

```diff
@@ -10,9 +10,9 @@ controlPlane:
     aws:
       rootVolume:
         iops: 4000
-        size: 500
+        size: 100
         type: io1
-      type: m4.xlarge
+      type: m5.xlarge
```

Comment:
Any specific reason why we're changing to m5 from m4?  We usually default to m4 for the cost savings!

Reply:
These are the new console defaults, approving!